pages/index.tsx

```diff
@@ -68,8 +68,12 @@ export default function Home({
       <Head>
         <title>{t('title')}</title>
         <link rel="icon" href="/claimstatus/favicon.ico" />
-        <link href="https://fonts.googleapis.com/css?family=Source Sans Pro" rel="stylesheet" />
-        <link href="/claimstatus/source-sans-pro-bold.ttf" rel="stylesheet" type="text/css" />
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossOrigin />
+        <link
+          href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+          rel="stylesheet"
+        />
```

Comment:
Note: the preconnect links come directly from Google Fonts and are for [improving font loading performance](https://www.cdnplanet.com/blog/faster-google-webfonts-preconnect/).

<img width="309" alt="Screen Shot 2021-08-12 at 11 58 44 AM" src="https://user-images.githubusercontent.com/67701/129253484-68cefe67-7b1a-4157-84f4-4eaa712fe2d8.png">